content/en/docs/tasks/network/validate-dual-stack.md

```diff
@@ -134,28 +137,46 @@ status:
   loadBalancer: {}
 ```
 
-Create the following Service with the `ipFamily` field set to `IPv6`.
+Create the following Service that explicitly defines `PreferDualStack` in `.spec.ipFamilyPolicy`. Kubernetes will assign both IPv4 and IPv6 addresses (as this cluster has dual-stack enabled) and select the `.spec.ClusterIP` from the list of `.spec.ClusterIPs` based on the address family of the first element in the `.spec.ipFamilies` array.
 
-{{< codenew file="service/networking/dual-stack-ipv6-svc.yaml" >}}
+{{< codenew file="service/networking/dual-stack-preferred-svc.yaml" >}}
 
-Validate that the Service gets a cluster IP address from the IPv6 address block. You may then validate access to the service via the IP and port.
-```
- kubectl get svc -l app=MyApp
-NAME         TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
-my-service   ClusterIP   fe80:20d::d06b   <none>        80/TCP    9s
+Validate that the Service gets a cluster IPs from the IPv4 and IPv6 address blocks using `kubectl describe`. You may then validate access to the service via the IPs and ports.
+
+{{< note >}}
+The `kubectl get svc` command will only show the primary IP in the `CLUSTER-IP` field.
```

Comment:
It's good to have this explanation, but should we also show the output of something like ` kubectl get svc -l app=MyApp` for additional clarity?

Reply:
Added